src/main/java/com/aws/greengrass/lifecyclemanager/GreengrassService.java

```diff
@@ -435,44 +434,49 @@ private List<GreengrassService> getHardDependers() {
         return dependers;
     }
 
-    public void addStateSubscriber(Subscriber s) {
-        lifecycle.getStateTopic().subscribe(s);
+    public void addStateListener(GlobalStateChangeListener s) {
```

Comment:
Good call, the only reason I see is that they're being used inside integ tests, but on a closer look these are not needed now since we're using the global listener which doesn't need a service instance to register/deregister. I'll just remove these